app/src/main/res/layout/register_screen.xml

```diff
@@ -8,7 +9,117 @@
             type="ip.team13.petowner.ui.register.RegisterViewModel" />
     </data>
 
-    <androidx.constraintlayout.widget.ConstraintLayout
+    <ScrollView
```

Comment:
Why do we need a `ScrollView` here? We should try to fit the content on the screen, `ConstraintLayout` does that

Reply:
Agree with that, but if the app is running on a small device screen, the content will not be shown right because we have lot of buttons and fields in it
But on normal or big screens, there is no difference, and it shouldn't be a problem

Reply:
There aren't that many fields, usually `ConstraintLayout` does a good job on chaining the elements with some margin between them for any screen size. We will also be targeting a high SDK version so there won't be too many phones in that category either. But let's talk about it later